apps/demo.namerank.io/app/results.tsx

```diff
@@ -0,0 +1,85 @@
+import { createClient } from "@namehash/namerank";
+import { Indicator } from "./indicator";
+
+const namerank = createClient({
+  namerankEndpoint:
+    "https://izzkysqb6d6qzhnpv4ybqyty2e0ktjwe.lambda-url.us-east-1.on.aws/namerank",
+});
+
+interface ResultsProps {
+  name: string;
+}
+
+export default async function Results({ name }: ResultsProps) {
+  const result = await namerank.inspectName(encodeURIComponent(name), {});
+  // const result = use(fetchNameRank(name));
+
+  if (!result.namerank || !result.namerank.analysis) {
+    return <p>=Please try again.</p>;
+  }
+
+  const topTokenization = result.namerank.analysis.top_tokenization || [];
+
+  return (
+    <div className="space-y-6">
+      <div>
+        <h3 className="text-lg font-semibold mb-2">Label For Analysis</h3>
+
+        <p className="flex items-center p-3 border border-gray-300 rounded mb-3 shadow-sm">
+          {result.namerank.analysis.inspection.label}
+        </p>
+      </div>
+
+      <div>
+        <h3 className="text-lg font-semibold mb-2">Top Tokenization</h3>
+        <div className="flex items-center p-3 border border-gray-300 rounded mb-3 shadow-sm">
+          {topTokenization.length > 0 ? (
+            topTokenization.map((token, index) => (
+              <span
+                key={index}
+                className={`rounded px-3 py-1 text-sm border font-semibold mr-2 ${
+                  token ? "bg-white border-gray-200" : "bg-gray-200 opacity-30"
+                }`}
+              >
+                {token || "..."}
+              </span>
+            ))
+          ) : (
+            <span className="text-gray-500">No tokenization available</span>
+          )}
+          {result.namerank.analysis.top_tokenization && (
+            <div className="ml-auto">
+              <Indicator value={result.namerank.purity_score} />
```

Comment:
For this "tokenization demo" my understanding is that all `Indicator` should be exclusively representing the `log_probability` and none should be related to purity score.

The purity score / interesting scores are great for the next enhancement to this app, but suggest it for a separate PR.

Reply:
Was a mistake, fixed.